realms/modules/search/views.py

```diff
@@ -1,10 +1,14 @@
 from flask import abort, g, render_template, request, redirect, Blueprint, flash, url_for, current_app
 from realms import search as search_engine
+from flask.ext.login import login_required, current_user
 
 blueprint = Blueprint('search', __name__)
 
-
```

Comment:
keep this line break to meet flake8 rules (2 empty lines in between top-level functions/classes)